src/zk.py

```diff
@@ -509,14 +513,14 @@ def acquire_lock(self, lock_type, allow_queue=False, timeout=None, read_lock=Fal
             raise ZookeeperException(f'Failed to acquire lock {lock_type}')
         logging.debug(f'Success acquire lock: {lock_type}')
 
-    def try_acquire_lock(self, lock_type=None, allow_queue=False, timeout=None, read_lock=False):
+    def try_acquire_lock(self, lock_type=None, allow_queue=False, timeout=None, read_lock=False, host_description=None):
```

Comment:
Parameter `host_description` was not added to `def acquire_lock` method, and it is almost the same wrapper around `_acquire_lock`. Shouldn't we add it for the sake of symmetry?